lilcom/lilcom_c_extension.c

```diff
@@ -110,8 +110,8 @@ static PyObject * compress_int16(PyObject * args, PyObject * keywds)
 {
   PyObject *input; // The input signal, passed as a numpy array.
   PyObject *output; // The output signal, passed as a numpy array.
-  int lpc_order = 5; // LPC Order defined in the core function (more information -> lilcom.h)
-  int conversion_exponent = 0; // Conversion Exponent defined in the core function (more information -> lilcom.h)
+  int lpc_order; // LPC Order defined in the core function (more information -> lilcom.h)
+  int conversion_exponent; // Conversion Exponent defined in the core function (more information -> lilcom.h)
 
```

Comment:
I remember that you stated these values shouldn't have default values in order not to face with problem made from defaults.